CHANGELOG.md

```diff
@@ -11,6 +12,36 @@
   - N/A
 
 # Edge
+## Breaking Changes
+
+The following Edge Management REST API Endpoints have breaking changes:
```

Comment:
should this be 0.26.x?

Reply:
Reasonable question. Below v1.0.0 breaking changes have no sway over versioning.

Reply:
We've generally been trying to bump the minor to indicate breaking changes